packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -188,8 +188,10 @@ export class AtomicRefineModal implements InitializableComponent {
 
   private renderFilters() {
     const isFacetElements = this.bindings.store.getFacetElements().length > 0;
+    const isAutomaticFacets =
+      this.bindings.engine.state.automaticFacetSet?.set !== undefined;
 
-    if (!isFacetElements) {
+    if (!isFacetElements && !isAutomaticFacets) {
```

Comment:
Wonder why this wasn't caught by tests? Any test gaps that can be fixed?

Reply:
I added corresponding tests !